src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -2,7 +2,42 @@
 from _gettsim.shared import policy_info
 
 
-def arbeitsl_geld_2_eink_m(  # noqa: PLR0913
+def arbeitsl_geld_2_eink_m(
+    _arbeitsl_geld_2_nettoeink_ohne_transfers_m: float,
+    kind_unterh_erhalt_m: float,
+    unterhaltsvors_m: float,
+    kindergeld_zur_bedarfdeckung_m: float,
+) -> float:
+    """SGB II income.
+
+    Relevant income according to SGB II. Does not include child benefit transfer
+    (Kindergeldübertrag).
```

Comment:
Can you explain why? Intuitively, Kindergeldübertrag seems like a component of income to me, so I don't quite understand the structure of not including it there but deducting it from the payments in the end.

Reply:
I was leaning towards not counting Kindergeldübertrag as income of the parent (but of the child) but now that I'm thinking about it, I cannot defend that point.

Will switch to something like

```python
def arbeitsl_geld_2_eink_m(...):
   return arbeitsl_geld_2_eink_ohne_kindergeldübertrag_m + kindergeldübertrag_m
```